Auth0/Auth0Authentication.swift

```diff
@@ -476,6 +476,10 @@ struct Auth0Authentication: Authentication {
                        telemetry: self.telemetry)
     }
 
+    func customTokenExchange(subjectTokenType: String, subjectToken: String, scope: String = "openid profile email", audience: String? = nil) -> Request<Credentials, AuthenticationError> {
```

Comment:
Default param values don't go here, they go in the respective extension method in Authentication.swift.